sync/util/etcd.go

```diff
@@ -15,22 +15,21 @@ var log = l.NewLogger()
 
 func CreateFromConfig(config *util.Config) (s sync.Sync, err error) {
 	syncType := config.GetString("sync", "etcd")
+	etcdServers := config.GetStringList("etcd", nil)
```

Comment:
I'm aware this is Not-Your-Code, and it's great you've refactored it a bit. Please remember to always add UTs for the changes you're making.